File: spark/src/main/scala/pl/com/sages/spark/conf/SparkConfBuilder.scala
package pl.com.sages.spark.conf

import org.apache.spark.SparkConf

/**
 * SparkConf builder
 */
object SparkConfBuilder {
  /*
  Local conf builder
   */
  def buildLocal(appName:String):SparkConf = {
    val sparkConf = new SparkConf()
      .setMaster("local[2]")
      .setAppName(appName)
    sparkConf
  }
}

File: core/src/main/java/pl/com/sages/spark/conf/SparkConfBuilder.java
package pl.com.sages.spark.conf;

import org.apache.spark.SparkConf;

/**
 * SparkConf builder
 */
public class SparkConfBuilder {
    public static SparkConf buildLocal(String appName) {
        return new SparkConf()
                .setMaster("local[2]")
                .setAppName(appName);
    }
}
